planning/behavior_velocity_planner/virtual-traffic-light-design.md

```diff
@@ -0,0 +1,288 @@
+### Virtual Traffic Light
+
+#### Role
+
+このモジュールは外部から指示が来たら、対応する仮想信号機の場所で止まります。
+このモジュールでは外部のオペレータと連携して交差点の調停や、インフラ連携してシャッターの手前で止まることなどが実現できます。
+Mediation at a intersection while linking to external operator, and planning a velocity of the stop. This module is used for example factory transportation or traffic priority is decided outside of Autoware.
```

Comment:
We've changed the word from `mediation` to `coordination`.

Reply:
Also, I feel we don't have to mention `factory transportation` here. It's not a role.